apps/linker/src/app/components/send-message-dialog/send-message-dialog.component.html

```diff
@@ -0,0 +1,30 @@
+<div>
+  <div mat-dialog-title>
+    <h2 class="page-subtitle">
```

Comment:
add css style for the page-subtitle class - in linker it is not defined

Reply:
Css class added.